### DISCUSSION TITLE
Connect to Hoywell - but how does it actually work

### DISCUSSION THREAD

**Discussion by tux-84** [tbnobody/OpenDTU#698] [Q&A] [answered]
Im planning to get on of [these](https://www.yuma.de/produkt/yuma-balcony-eco-l2/) solar panels for my balcony. Since I'm already running a quite elaborate setup of HassIO I'm also interested in integrating the HM600. So I stumbled over this project and the esp32 stuff is nothing new to me BUT: How to I actually connect these two boards to the HM600 in the end? I've not found any photos of this procedure. I also took a look at the thread on mikrokontroller.net but there's lot's and lot's of technobabble I'm not interested it and the thread is cluttered like hell. So how does this "thing" actually work? Can someone explain this to me?

**Comment by Solarteur** [comment, marked as answer]
OpenDTU is the "bridge" from a Hoy**miles** inverter (**HM**-xxx) to the local WLAN. A DTU communicates with one or more inverters via the nRF24L01+ module (2,4 GHz). The communication originates from the inverter - DTU makes the request and expects a response from the inverter. The DTU must know the serial number(s) of all inverters. These are entered as part of the setup process.

**Reply by tux-84** [reply]
Thanks for your explanation. I think this short summary should be added to the ReadMe of OpenDTU right before all the setup and flashing process is explained. I can't imagine that questions like mine didn't have come up here already.
